gateway/ln-gateway/src/lib.rs

```diff
@@ -514,6 +514,10 @@ impl Gateway {
                                             }
                                         }
 
+                                        self.register_clients_timer(&mut htlc_task_group).await;
+                                        self.load_clients().await;
```

Comment:
This changed since your last rebase. We don't want to re-load the clients and re-register the timer loop on every connection to the lightning node anymore.

Reply:
removed in latest commit